plugins/base/src/main/resources/dokka/styles/style.css

```diff
@@ -100,62 +76,421 @@ html ::-webkit-scrollbar-thumb {
+}
+
+.no-js .filter-section {
+    display: none;
+}
+
+@media (min-width: 760px) {
+    .filter-section {
+        padding: 5px 0 5px;
+    }
+}
+/* --- Navigation controls --- */
+.navigation-controls {
+    display: flex;
+}
+
+@media (min-width: 760px) {
+    .navigation-controls {
+        align-items: center;
+    }
+}
+
+.no-js .navigation-controls {
+    display: none;
+}
+
+/* --- Navigation THEME --- */
+.navigation-controls--search {
+    display: inline-flex;
+    font-size: 0;
+    line-height: 0;
+}
+
+.navigation-controls--theme {
+    display: block;
+    border-radius: 50%;
+    background-color: inherit;
+    margin-left: 4px;
+    padding: 0;
+    border: none;
+    cursor: pointer;
+    font-size: 0;
+    line-height: 0;
+}
+
+.navigation-controls--theme::before {
+    height: 40px;
+    width: 40px;
+}
+
+.navigation-controls--theme:hover {
+    background: var(--white-10);
+}
+
+.navigation-controls--theme::before {
+    display: block;
+    content: url("../images/theme-toggle.svg");
+}
+
+@media (max-width: 759px) {
+    .navigation-controls--theme {
+        display: none;
+    }
+}
+/* /--- Navigation THEME --- */
+
+.navigation .platform-selector:not([data-active]) {
+    color: #fff;
+}
+/* /--- Navigation controls --- */
+/* /--- Navigation styles --- */
+
+/* --- Layout styles --- */
+
+#container {
+    display: flex;
+    flex: 1 1 auto;
+    min-height: 0; /* full height exclude header */
+}
+
+#container > .sidebar, #container > #main {
+    overflow: auto;
+}
+
+#main {
+    display: flex;
+    flex-direction: column;
+    flex: 1 1 0; /* full width, but no affects for sidebar */
+}
+
+.sidebar {
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+    border-right: 1px solid var(--border-color);
+    width: var(--sidebar-width);
+}
+
+.no-js .sidebar {
+    display: none;
+}
+
+/*.theme-dark .sidebar {
+    box-shadow: inset 0 1px 0 0 hsl(0deg 0% 100% / 30%);
+}*/
+
+@media (max-width: 759px) {
+    #container {
+        position: relative;
+    }
+
+    .sidebar {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        box-sizing: border-box;
+        background: var(--background-color);
+        margin-left: calc(-1 * var(--sidebar-width));
+        transition: margin .2s ease-out;
+        z-index: 4;
+    }
+
+    .sidebar.open {
+        margin-left: 0;
+    }
+
+    .sidebar.open ~ #main .navigation-controls--search {
+        display: none;
+    }
+
+    .sidebar.open ~ #main .menu-toggle {
+        margin-left: var(--sidebar-width);
+    }
+}
+
+.sidebar--inner {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 16px;
+    padding-top: 22px;
+    padding-bottom: 16px;
+}
+/* /--- Layout styles --- */
+
+/* --- Main Content styles --- */
+.main-content {
+    padding-bottom: var(--bottom-spacing);
+    margin-left: auto;
+    margin-right: auto;
+    max-width: var(--max-width);
+    width: 100%;
+    z-index: 0;
+}
+
+.main-content > * {
```

Comment:
Ouch, I didn't expect that.
I see it was there before your changes — no questions for you.
Using * looks weird to me with these rules. I prefer explicit content control.